static/js/protein/chart.ts

```diff
@@ -219,6 +240,76 @@ function addYLabel(
     .text(labelText);
 }
 
+export function drawTissueLegend(id: string, data: ProteinStrData[]): void {
+  // checks if the data is empty or not
+  if (_.isEmpty(data)) {
+    return;
+  }
+  const svg = d3
+    .select("#tissue-score-legend")
+    .append("svg")
+    .attr("width", GRAPH_WIDTH_XL)
+    .attr("height", GRAPH_HEIGHT_XS);
+  const data0 = d3.keys(TISSUE_LEGEND_DICT);
+  // slicing the dictionary in half to display the legend in two rows
+  const data1 = data0.slice(0, 8);
+  const data2 = data0.slice(8, 15);
+  // creating circles for the first row in legend
+  svg
+    .selectAll("mydots1")
```

Comment:
- since the dots on the first row and the second row should be styled the same way, don't need to distinguish between "mydots1" and "mydots2"
- when naming components and variables, adding possessive adjectives (eg. my) is unnecessary. Here, it can just be "dots" or "legend-dots"

Reply:
Done!